python/lsst/afw/display/rgb.py

```diff
@@ -257,12 +259,12 @@ def __init__(self, minimum, dataRange, Q=8):
                 Q = Qmax
 
         if False:
-            self._slope = self._uint8Max/Q # gradient at origin is self._slope
+            self._slope = self._uint8Max/Q  # gradient at origin is self._slope
```

Comment:
Yes, this was caused by an interactive rebase where I backed out an `old_div` change but ended up retaining the whitespace fix by mistake.